flybywiresim/a32nx
# feat: flyPad ATC view filter for controller frequencies

Fixes #7483

## Summary of Changes
Provides a free text search on controller call signs and a filter for the different Controller type to make it easier to find the correct stations when a lot of stations are available. 

Also adds a listener to SimpleInput to use Enter to leave/unfocus the input field. 

## Screenshots (if necessary)
![image](https://user-images.githubusercontent.com/16833201/189909024-6e25c025-9a4a-44d3-bebe-f6153929c2ff.png)
![image](https://user-images.githubusercontent.com/16833201/189909086-a15b937b-69c4-4c5e-8da4-6c4c7f165bb4.png)
![image](https://user-images.githubusercontent.com/16833201/189909113-be23590e-99b4-4370-bfa1-c48cfb51160d.png)


Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
- Make sure the flyPad ATC page works as before
- Test at an airport with lots of VATSIM/IVAO activity 
- Test CallSign Search and the filters and check if these match
- Check layout when many stations are available or station info is very long. 
- Check all Input fields of the EFB - pressing enter now leaves/unfocuses the input field (as long as mouse pointer is not on the field). This could have an impact on other things and cause issues. Please test. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
